mod/deckorate_attribution/spec/attribution_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe Card::Set::Type::Reference do
+  def card_subject_name
+    "Reference 0000001"
+  end
+  check_views_for_errors
+
+  context "when a dataset with an alteration." do
+    let(:card) { Card.fetch("Reference 0000001") }
+
+    it "shows who did the adaptation in Rich Text." do
+      expect(card.format.render(:rich_text_attrib)).to match(/adaptation by Joe User/)
```

Comment:
This is synonymous with:
` expect_view(:rich_text_attrib).to match(/adaptation by Joe User`